test/decoders/test_simple_video_decoder.py

```diff
@@ -4,6 +4,7 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
+import numpy
```

Comment:
numpy is now a test-time dependency. Do we need to update any docs?

Reply:
It's already a test time dependency. :) See https://github.com/pytorch/torchcodec/blob/1fffd024cbf12c3e6116f1555c676ff7adc86d40/test/decoders/test_video_decoder_ops.py#L13 and https://github.com/pytorch/torchcodec/blob/1fffd024cbf12c3e6116f1555c676ff7adc86d40/test/utils.py#L9